docs/types/value-types.rst

```diff
@@ -305,8 +305,8 @@ Since byzantium ``staticcall`` can be used as well. This is basically the same a
 
 All three functions ``call``, ``delegatecall`` and ``staticcall`` are very low-level functions and should only be used as a *last resort* as they break the type-safety of Solidity.
 
-The ``gas`` option is available on all three methods, while the ``value`` option is not
-supported for ``delegatecall``.
+The ``gas`` option is available on all three methods, while the ``value`` option is only available
+on ``call``.
```

Comment:
Maybe we should also explain why? I.e. `staticcall` cannot modify state and `delegatecall` works on current contract's state so sending anything to the other contract would be pointless.

Reply:
The reason is just that the opcode doesn't have a field for the value :) 

I think I would skip explaining the reasoning here.

Reply:
ok. fair enough.